app/Filament/Resources/GameResource/Pages/Edit.php

```diff
@@ -5,7 +5,9 @@
 namespace App\Filament\Resources\GameResource\Pages;
 
 use App\Filament\Concerns\HasFieldLevelAuthorization;
+use App\Filament\Enums\ImageUploadType;
 use App\Filament\Resources\GameResource;
+use App\Filament\Resources\NewsResource\Actions\ProcessUploadedImageAction;
```

Comment:
We should probably lift this out of NewsResource. Maybe it can just live the an `App\Filament\Actions` namespace?

Reply:
done.